test_check_if_anagrams.py

```diff
@@ -1,6 +1,12 @@
 def are_anagrams(str1, str2):
-  # Remove the `return None` code below and add your own code.
-  return None
+  str1 = str1.lower()
+  str2 = str2.lower()
+  for x in range (0, len(str1)):
+    if str2.find(str1[x])!=-1:
+      str2 = str2.replace(str1[x], "", 1)
```

Comment:
That line removes the first instance of any character from str1 that is found is found in str2.  The whole loop will go through each character in str1 and see if there is a match in str2, then remove that match.  If the two strings aren't anagrams, eventually there will not be a match in line 5 so it will return False.